src/libraries/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoCurrencyNegativePattern.cs

```diff
@@ -13,6 +13,11 @@ public static IEnumerable<object[]> CurrencyNegativePattern_TestData()
         {
             yield return new object[] { NumberFormatInfo.InvariantInfo, new int[] { 0 } };
             yield return new object[] { CultureInfo.GetCultureInfo("bg-BG").NumberFormat, new int[] { 0, 8 } };
+            if (PlatformDetection.IsNotUsingLimitedCultures)
+            {
+                // ICU for mobile / browser do not contain these locales
+            	yield return new object[] { CultureInfo.GetCultureInfo("luy-KE").NumberFormat, new int[] { 16 } };
```

Comment:
>Info("luy-KE").Numbe [](http://example.com/codeflow?start=63&length=20)

Please avoid adding more culture names t this test cases. Instead a dd test case which create writable culture and overwrite the currency negative pattern then format a number and validate. Something like:

```C#
            CultureInfo ci = new CultureInfo("en-US");
            ci.NumberFormat.CurrencyNegativePattern = 16;
            double number = -16.15;
            // validate the formatted number number.ToString("C", ci)}
```

Reply:
and the new test can be marked with the attribute
```
        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsIcuGlobalization))]
````

Reply:
I think `double.ToString` tests are inside Double Tests.